README.md

```diff
@@ -104,3 +104,10 @@ instructions on using `pep8`, `pylint` and `pyflakes` to check Python code.
 ## Documentation
 
 See the [`docs`](docs) directory for setup and configuration of Markdown linting.
+
+
+## Copyright Headers
+
+All Mobify source files should contain copyright headers at the beginning of the file.
```

Comment:
@johnboxall "*All*" or just source we intend to ship outside our control (ie. to the browser/device). I wouldn't anticipate server code needs these?